core/src/validator.rs

```diff
@@ -969,12 +969,30 @@ impl Validator {
                     )),
                     Some((&staked_nodes, &identity_keypair.pubkey())),
                 );
-                Arc::new(connection_cache)
+
+                let connection_cache_fwd = ConnectionCache::new_with_client_options(
+                    "connection_cache_tpu_fwd_quic",
+                    1,
+                    None,
+                    Some((
+                        &identity_keypair,
+                        node.info
+                            .tpu(Protocol::UDP)
```

Comment:
If this is for QUIC then probably we better use `.tpu(Protocol::QUIC)`.
I know you just need the IP address, but the `tpu_quic` and `tpu_udp` might have different IP addresses, and with the new contact-info, a node might not even specify `tpu_udp`.

Reply:
This is just copied code from above -- I will defer it to another PR.

Reply:
Deferred to another PR to reduce impact